authorization/src/main/java/life/qbic/authorization/application/AppContextProvider.java

```diff
@@ -0,0 +1,21 @@
+package life.qbic.authorization.application;
+
+/**
+ * <b>App Context Provider</b>
+ * <p>
+ * Provides some utility methods to create navigation targets within the application.
+ *
+ * @since 1.0.0
+ */
+public interface AppContextProvider {
+
+  /**
```

Comment:
This looks similiar to the `PasswortResetLinkSupplier` class maybe this could be streamlined?

Reply:
Yes exactly, however this was my first effort to refactor it. The reset link supplier and confirmation link supplier would be also implemented in the app module and the interface extended accordingly.